src/stories/Accordion.stories.mdx

```diff
@@ -0,0 +1,217 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks'
+import { Accordion, AccordionDetail, AccordionHeader, AccordionsWrapper } from '../components'
+import { ThemeProvider } from '../theme'
+
+<Meta title='Data Display/Accordion' component={Accordion} />
+
+# Accordion
+
+O componente `Accordion` permite que uma área seja encolhida e/ou expandida. Deve ser utilizado para agrupar ou ocultar regiões complexas, mantendo a página limpa. A Sua utilização baseia-se na importação e utilização de 4 componentes. O `AccordionsWrapper` deve ser o container em volta de todos os Accordions, cada item consiste (Nesta Ordem) de um componente `Accordion` que você deve colocar como filhos dele o componente `AccordionHeader` que define o seu título e o conteúdo (children) do Accordion sendo definido no componente `AccordionDetail`.
+
+<Canvas>
+  <Story
+    name='Base'
+    parameters={{
+      design: {
+        type: 'figma',
+        url: 'https://www.figma.com/file/O3bKxIcsj2rc1FNIRclJyT/Saturn-System?node-id=560%3A0'
+      }
+    }}
+  >
+    <ThemeProvider>
+      <AccordionsWrapper>
+        <Accordion>
+          <AccordionHeader title='Lorem Ipsum' />
+          <AccordionDetail>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis sed ligula at lorem pretium pulvinar sodales
+            sed eros. Donec a risus a mauris pretium scelerisque.
+          </AccordionDetail>
+        </Accordion>
+        <Accordion style={{ backgrounColor: '#f0f' }}>
```

Comment:
`backgroundColor` tá com um erro de grafia, além disso esse style in-line acredito que não devia estar aí.

Reply:
realmente não era pra estar, estava usando durante o desenvolvimento e esqueci de arrumar